ckanext/archiver/tasks.py

```diff
@@ -123,12 +111,10 @@ class CkanError(ArchiverError):
     pass
 
 
-def update_resource(ckan_ini_filepath, resource_id, queue='bulk'):
+def update_resource(resource_id, queue='bulk'):
```

Comment:
can you add deprecated companion methods with old signature so transactions in flight don't get dropped on upgrade on all task methods please.

Reply:
Hmm. Python can't overload the function like Java can, and since the parameter to be deleted is the first one and we're using positional arguments, we can't just supply a default value for it. Might be able to sort it out by switching to keyword arguments.